pandas/tests/arrays/masked/test_arrow_compat.py

```diff
@@ -184,6 +184,13 @@ def test_pyarrow_array_to_numpy_and_mask(np_dtype_to_arrays):
     tm.assert_numpy_array_equal(mask, mask_expected_empty)
 
 
+def test_from_arrow_null(data):
```

Comment:
Can you add a test for chunked arrays with different lengths?

Reply:
Done.